include/fluent-bit/flb_task.h

```diff
@@ -50,11 +50,29 @@
 #define FLB_TASK_SET(ret, task_id, out_id)              \
     (uint32_t) ((ret << 28) | (task_id << 14) | out_id)
 
+#define FLB_TASK_ROUTE_HOOK_RETURN  1
+#define FLB_TASK_HOOK_ONCE          1
```

Comment:
same value as the above. This is intended?

Reply:
Yes. These are for different "enums" but there are no enums in this